mitodl/open-discussions
# New setting to specify if only course images should be uploaded

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/zagaran/ocw-data-parser/issues/7

#### What's this PR do?
Adds a new boolean setting `OCW_UPLOAD_IMAGE_ONLY` (default=False).  If True, the OCW parser will upload only the course image to S3, otherwise it will upload all course files. 

#### How should this be manually tested?
- In your .env file, set `OCW_UPLOAD_IMAGE_ONLY=True`
- In a shell, run `get_ocw_data()`.  As the courses are imported you should see only images being uploaded in the logs, no other files such as pdf or zip:
```
[2019-02-11 15:46:08] INFO 1 [course_catalog.tasks] tasks.py:91 - [8c2288eddda9] - Syncing: PROD/16/16.412J/Spring_2016/16-412j-cognitive-robotics-spring-2016/ ...
[2019-02-11 15:46:24] INFO 1 [ocw_data_parser.ocw_data_parser] ocw_data_parser.py:373 - [8c2288eddda9] - Uploaded 54d1a777a7666799ad583f802b147c0d_16-412js16.jpg
[2019-02-11 15:46:24] INFO 1 [ocw_data_parser.ocw_data_parser] ocw_data_parser.py:378 - [8c2288eddda9] - Uploaded 54d1a777a7666799ad583f802b147c0d_16-412js16.jpg
```
